src/prettify.js

```diff
@@ -1740,3 +1741,14 @@ var prettyPrint;
     });
   }
 })();
+
+// npm require support
+if (module && module.exports) module.exports = {
```

Comment:
Won't if(module) throw an exception if there is no global named "module".

Maybe if (typeof module !== 'undefined' && module.exports)